requirements.txt

```diff
@@ -12,7 +12,7 @@ Jinja2==2.10
 pbr==5.1.1
 pexpect==4.6.0
 psutil==5.4.6; sys_platform!="win32" and sys_platform!="cygwin"
-PyYAML==3.13
+PyYAML>=5.1  # MIT
```

Comment:
Why MIT? Also, this might deserve a major release.

cc @decentral1se

Reply:
@webknjaz The comment is only informative, I have seen documenting licenses of dependencies as comments as a common practice in OpenStack and even outside.

I think it does not hurt to start documentig licenses this way.

Reply:
I doubt we need a major release only for bumping some requirments. As long we do not break our API everything would count as minor.

Reply:
The API remains unchanged, so think we're looking at a minor release here. Shall we add a change log entry for this? For the comments: it may confuse contributors to see comments with licenses on some dependencies and not on others (is this optional? What is this for? Should I do this?), so I'd rather just avoid this additional practice for our dependency management on this project.

Let's merge this soon!

Reply:
changelog entry: yes
merge: move to `setup.cfg`

Reply:
I would refrain from migrating more into setup.cfg until packaging is fixed. I am considering proposing revert of the famous packaging change from last week https://github.com/ansible/molecule/issues/1966. 

I think it was too early as it was not tested enough.

Reply:
It's not broken. Your use-case is not supported and you didn't provide steps to reproduce so it doesn't count. It works as long as you use it as designed.